resources/_gen/assets/js/js/search.js_b4bb3e7b21b43e920b7c367ff787d383.content

```diff
@@ -27,7 +27,7 @@ limitations under the License.
                     }
 
                     var query = $(this).val();
-                    var searchPage = "search/?q=" + query;
+                    var searchPage = "https://www.opencue.io/search/?q=" + query;
```

Comment:
Is it bad for `www.opencue.io` to be hardcoded here, since it will sometimes be deployed to other URLs?

Reply:
Great question, and I think this is fine.

We recently updated line 1 of config.toml:

```toml
baseURL = "https://www.opencue.io"
```

This parameter is used by Hugo at build time to configure a bunch of aspects of the site, including the RSS feed, robots.txt, and search.

If someone deploys the site to another URL, they also need to update `baseURL` in their copy of config.toml. Otherwise the site won't really be 'fully' functional.

With regards to search, they also would need to register and configure Google Custom Search Engine for their domain.

Hope that's clear, but let me know if you have further questions.

Reply:
That makes sense, thanks.